my-profile-webapp/src/main/webapp/partials/countries.html

```diff
@@ -0,0 +1,251 @@
+<select id="countries" class="form-control" required ng-model="contact.addresses[0].country">
```

Comment:
With the current json structure address.country wouldn't work - needs to be addresses[0].country right?